dojo/templates/dojo/view_test.html

```diff
@@ -604,12 +604,15 @@ <h4 class="has-filters"> Findings ({{findings.total_count}}) <small>{{ test.id|g
                             <b>Push to JIRA</b> <input id="id_push_tojira" name="push_to_jira" type="checkbox" alt="Select to push to JIRA" title="Push_To_Jira"/><br/>
 
                         {% endif %}
+                        <label style="display: block">Notes</label>
+                        {{ bulk_edit_form.notes }}
                         <label style="display: block">Tags</label>
                         {% comment %}
                             Quick hack to make bulk edit work without refactoring the whole bulk edit form
+                            {{ bulk_edit_form.media.css }}
```

Comment:
Why did you move these 2 lines?

Reply:
To be honest, I have no idea what those two lines are doing, in `finding_list_snippet.html` this lines are under the comment. So I simply tested with and without this two lines, I can't determinate the difference, so I decide to turn it off but preserve.